src/components/OHAEditor/index.js

```diff
@@ -276,7 +276,27 @@ export default ({
                 changeSingleSampleOHA(
                   setIn(singleSampleOHA, ["taskOutput", relativeIndex], output)
                 )
-                onChangeOHA(newOHA)
+
+                let sendOnChange = true
+                if (
+                  oha.taskOutput &&
+                  newOHA.taskOutput &&
+                  oha.taskOutput[singleSampleOHA.sampleIndex] !== null
+                ) {
+                  const o1 = oha["taskOutput"][singleSampleOHA.sampleIndex]
+                  const o2 = newOHA["taskOutput"][singleSampleOHA.sampleIndex]
```

Comment:
something very useful about seamless immutable objects is you can usually just do `o1 === o2`

Reply:
that compares length ?